crates/ty_python_semantic/src/types.rs

```diff
@@ -9675,6 +9699,9 @@ impl<'db> KnownBoundMethodType<'db> {
                     Some(KnownClass::Bool.to_instance(db)),
                 )))
             }
+            KnownBoundMethodType::PathOpen => {
+                Either::Right(std::iter::once(Signature::todo("`Path.open` return type")))
```

Comment:
This is overly aggressive, because now we replace the whole signature with `(*args, **kwargs) -> @Todo`, i.e. we also have false negatives for wrong `Path.open` calls, and return a `@Todo` type for every call to `Path.open`. But manually constructing a overload set for `Path.open` seems daunting.